File: 단계별로 풀어보기/정렬/2750/수 정렬하기.py
import sys
        
def init():
    N = int(sys.stdin.readline())
    num =[]
    for i in range(N):
        num.append(int(sys.stdin.readline()))

    for j in range(N):
        minimum = num[j]
        for i in range(j,N):
            if minimum > num[i]:
                temp = minimum
                minimum = num[i]
                num[i] = temp
        num[j] = minimum

    for i in range(N):
        sys.stdout.write('{}\n' .format(num[i]))
                         
init()

File: 단계별로 풀어보기/정렬/2750/수 정렬하기.cpp
#include <iostream>
using namespace std;

int main(){
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    
    int N, minimum, temp;
    int num[1000] = {0};
    
    cin >> N;
    
    for(int i=0; i<N; i++)
        cin >> num[i];
        
    for(int j=0; j<N; j++){
        minimum = num[j];
        for(int i=j; i<N; i++){
            if(minimum > num[i]){
                temp = minimum;
                minimum = num[i];
                num[i] = temp;
            }
        }
        num[j]=minimum;
    }
    
    for(int i=0; i<N; i++){
        cout << num[i] << '\n';
    }
    
    return 0;
}
